samples/wifi/shell/src/wifi_main.c

```diff
@@ -15,4 +23,14 @@ void main(void)
 			       NRF_CLOCK_HFCLK_DIV_1);
 #endif
 	printk("Starting %s with CPU frequency: %d MHz\n", CONFIG_BOARD, SystemCoreClock/MHZ(1));
+
+#ifdef CONFIG_NET_L2_OPENTHREAD
```

Comment:
I guess we can remove this conditional flag...i.e. make this applicable by default.